### PR TITLE
Docs: claim assets on testnet v1

### PR DESCRIPTION
**Pull request by ShookLyngs**
### Description
Added a tutorial about how to claim faucet on Godwoken testnet_v1.
Make sure to fulfill the checklist before merging.

### Checklist
- [x] Merge #86
- [x] Change base branch to `main`

### CODE REVIEW

File: docs/claimFaucet.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+- Bridge assets from Goerli to Godwoken
+- Claim CKB via Account-Faucet
+
+## Obtain your layer1/layer2 address
+
+As a first step, you need to know what your layer1/layer2 addresses are, so then you can claim faucet to them.
+
+Godwoken Bridge is an asset bridge to help us transfer assets between layer1 and layer2, it also provides a clean UI for you to check your layer1/layer2 addresses. You can follow these steps to connect to Godwoken Bridge:
+
+1. Open [Godwoken Bridge](https://testnet.bridge.godwoken.io/)
```
Comment (Flouse) [inline docs/claimFaucet.md:24]:
```suggestion
1. Open [Godwoken Bridge (testnet)](https://testnet.bridge.godwoken.io/)
```

File: docs/claimFaucet.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+| `L1 Wallet Address`  | Your layer1 address, this is a CKB omni-lock address, you should use this address when claiming for layer1 assets                                     |
+| `L1 Deposit Address` | Your layer1 deposit address, you can transfer specific amount of capacity/sUDT to this address, and Godwoken will collect them to your layer2 account |
+| `Ethereum Address`   | Your layer2 address, you should use this address when claiming for Goerli assets, or when you make actions on Ethereum networks                       |
+| `L1 Balance`         | Your layer1 balance, it means how much CKB your `L1 Wallet Address` has on CKB layer1, you can deposit your `L1 Balance` to layer2                    |
+| `L2 Balance`         | Your layer2 balance, it means how much pCKB your **`Ethereum Address`** has on Godwoken layer2, you can withdraw your `L2 Balance` to layer1          |
+
+If you’re new to Godwoken, it’s normal that your `L1 Balance` and `L1 Balance` are both `0`. You can follow the tutorials below to claim free assets to your layer1/layer2 accounts.
```
Comment (Flouse) [inline docs/claimFaucet.md:37]:
```suggestion
If you’re new to Godwoken, it’s normal that your `L1 Balance` and `L2 Balance` are both `0`. You can follow the tutorials below to claim free assets to your layer1/layer2 accounts. 
```

File: docs/claimFaucet.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+
+:::tip
+For more details on Godwoken Bridge, please visit: [Deposit & Fast Withdrawal to Godwoken v1](fastWithdrawal.md).
+:::
+
+## Claim Layer1 assets
+
+### Claim CKB
+
+On the CKB layer1 testnet, you can claim CKB using the Nervos Aggron Faucet, a official faucet site provided by Nervos. On Nervos Aggron Faucet, each layer1 address can claim 10,000 CKB once every 24 hours.
```
Comment (Flouse) [inline docs/claimFaucet.md:47]:
```suggestion
On the CKB layer1 testnet, you can claim CKB using the Nervos Faucet, an official faucet site provided by Nervos. On Nervos Faucet, each layer1 address can claim 10,000 CKB once every 24 hours.
```

Comment (Flouse) [reply docs/claimFaucet.md:47]: Pudge is the successor guardian of the Aggron.
> https://github.com/nervosnetwork/rfcs/blob/5ccfef8a5e51c6f13179452d3589f247eae55554/rfcs/0037-ckb2021/0037-ckb2021.md#ckb2021-timeline

**Review by Flouse — CHANGES_REQUESTED** [review]

File: docs/claimFaucet.md [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+
+### Claim pCKB
+
+If you want to claim pCKB to your layer2 account, there are two recommended methods.
+
+The first method is if you only need a small amount of pCKB, you can visit [Test pCKB Faucet](https://faucet.godwoken.com/), and then click the “request 10 pCKB from faucet” button to claim 10 pCKB to your layer2 account, on the way it will ask you to connect to your wallet, you should just follow the instructions and the whole process will complete soon.
+
+However, if you want to claim a large amount of pCKB, the second method is more recommended: [Nervos Aggron Faucet](https://faucet.nervos.org/). 
+
+Nervos Aggron Faucet is an official faucet site provided by Nervos. On Nervos Aggron Faucet, each layer1 address can claim 10,000 CKB once every 24 hours. The site only provides CKB, but you can claim funds using your `L1 Deposit Address`, so it's like Nervos Aggron Faucet depositing capacity to your layer2 account.
```
Comment (Flouse) [inline docs/claimFaucet.md:85]:
```suggestion
```

File: docs/claimFaucet.md [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+## Claim Layer1 assets
+
+### Claim CKB
+
+On the CKB layer1 testnet, you can claim CKB using the Nervos Aggron Faucet, a official faucet site provided by Nervos. On Nervos Aggron Faucet, each layer1 address can claim 10,000 CKB once every 24 hours.
+
+Follow these steps to claim CKB to your layer1 account:
+
+1. Copy your `L1 Wallet Address` on [Godwoken Bridge](https://testnet.bridge.godwoken.io/)
+2. Open [Nervos Aggron Faucet](https://faucet.nervos.org/)
```
Comment (Flouse) [inline docs/claimFaucet.md:52]:
```suggestion
2. Open [Nervos Faucet](https://faucet.nervos.org/)
```

File: docs/claimFaucet.md [hunk 6]
```diff
@@ -0,0 +1,163 @@
...
+Nervos Aggron Faucet is an official faucet site provided by Nervos. On Nervos Aggron Faucet, each layer1 address can claim 10,000 CKB once every 24 hours. The site only provides CKB, but you can claim funds using your `L1 Deposit Address`, so it's like Nervos Aggron Faucet depositing capacity to your layer2 account. 
+
+You can follow the below steps to try it out:
+
+1. Copy your `L1 Deposit Address` on [Godwoken Bridge](https://testnet.bridge.godwoken.io/)
+2. Open [Nervos Aggron Faucet](https://faucet.nervos.org/)
+3. Paste `L1 Deposit Address` in the page input
+4. Click “Claim” button next to the page input to claim
+
+After all above steps are completed, Godwoken will find the CKB cell that Nervos Aggron Faucet just sent you, and convert it into your layer2 capacity. So in conclusion, every CKB cell Nervos Aggron Faucet transfers to your `L1 Deposit Address` is a deposit cell.
```
Comment (Flouse) [inline docs/claimFaucet.md:94]:
```suggestion
After all above steps are completed, Godwoken will find the CKB cell that Nervos Faucet just sent you, and convert it into your layer2 account. So in conclusion, the CKB cell Nervos Faucet transfers to your `L1 Deposit Address` is a deposit cell. 
```

File: docs/claimFaucet.md [hunk 7]
```diff
@@ -0,0 +1,163 @@
...
+You can follow the below steps to try it out:
+
+1. Copy your `L1 Deposit Address` on [Godwoken Bridge](https://testnet.bridge.godwoken.io/)
+2. Open [Nervos Aggron Faucet](https://faucet.nervos.org/)
+3. Paste `L1 Deposit Address` in the page input
+4. Click “Claim” button next to the page input to claim
+
+After all above steps are completed, Godwoken will find the CKB cell that Nervos Aggron Faucet just sent you, and convert it into your layer2 capacity. So in conclusion, every CKB cell Nervos Aggron Faucet transfers to your `L1 Deposit Address` is a deposit cell. 
+
+How Godwoken deposit works with Nervos Aggron Faucet:
```
Comment (Flouse) [inline docs/claimFaucet.md:96]:
```suggestion
How Godwoken deposit works with Nervos Faucet:
```

File: docs/claimFaucet.md [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+First step, let’s bridge assets from Goerli testnet to CKB layer1:
+
+1. Copy your `L1 Wallet Address` on [Godwoken Bridge](https://testnet.bridge.godwoken.io/)
+2. Open [Force Bridge](https://testnet.forcebridge.com/bridge/Ethereum/Nervos)
+3. Select an asset you’d like to bridge, then input the amount you want to bridge
+4. Paste `L1 Wallet Address` in the “Recipient” input
+5. Click “Approve” button to allow Force Bridge to bridge the selected asset
+6. When the approval is completed, click “Bridge” button to bridge the selected asset to CKB layer1
+
+When the above process is complete, you should be able to see the bridged assets on Godwoken Bridge. Now you can deposit the bridged assets from CKB layer1 to Godwoken layer2, by following the below steps:
```
Comment (Flouse) [inline docs/claimFaucet.md:149]:
```suggestion
When the above process is completed, you should be able to see the bridged assets on Godwoken Bridge. Now you can deposit the bridged assets from CKB layer1 to Godwoken layer2, by following the below steps:
```

File: docs/claimFaucet.md [hunk 9]
```diff
@@ -0,0 +1,163 @@
...
+2. Enter the amount of deposit CKB in the “Deposit” input
+3. Click “Select Token” and then select the bridge asset
+4. Enter the amount of deposit token in the “sUDT (optional)” input
+5. Click “Deposit” button to deposit the bridge asset to Godwoken layer2
+
+Congratulations, the whole process of bridging Goerli assets to Godwoken is complete. When the above process is all done, you should be able to see your layer2 assets’ balance in the withdrawal page of Godwoken Bridge.
+
+## Claim CKB via Account-Faucet
+
+If you are tired of claiming faucet on Nervos Pudge Faucet every day, or if you have a lot of accounts that nees to claim faucet, maybe you should try out Account-Faucet, a command-line tool that can help you claim CKB on layer1/layer2, but easier.
```
Comment (Flouse) [inline docs/claimFaucet.md:161]:
```suggestion
If you are tired of claiming faucet on Nervos Faucet every day, or if you have a lot of accounts that need to claim faucet, maybe you should try out `Account-Faucet`, a command-line tool that can help you claim CKB on layer1/layer2, but easier.
```

File: docs/claimFaucet.md [hunk 10]
```diff
@@ -0,0 +1,163 @@
...
+## Overview
+
+If you want to test the functionality of Godwoken on testnet, but you don't have any asset available, you can follow this tutorial to claim assets to your testnet accounts.
+
+In this section, you will learn how to:
+
+- Obtain your layer1/layer2 address
+- Claim layer1/layer2 assets
+- Bridge assets from Goerli to Godwoken
+- Claim CKB via Account-Faucet
```
Comment (Flouse) [inline docs/claimFaucet.md:16]:
```suggestion
- Claim CKB via Account-Faucet CLI
```

**Review by Flouse — CHANGES_REQUESTED** [review]

File: docs/claimFaucet.md (hunk 4 above)
Comment (ShookLyngs) [reply docs/claimFaucet.md:85]: I think we should keep this part of content, because readers might skip the #claim-ckb part.

Comment (ShookLyngs) [reply docs/claimFaucet.md]: I think we should keep this section as it explains a bit about Nervos faucets. I've edited this part and tried to make it more reasonable, please take a look.

**Review by Flouse — APPROVED** [review]

**Pull request merged**